src/App.css

```diff
@@ -1,5 +1,6 @@
-.stack {
-  inline-size: 90%;
-  max-inline-size: 340px;
-  margin: var(--n-space-xl) auto;
+#root {
+  max-width: 1280px;
```

Comment:
Could this also be converted to use logical properties instead?

Reply:
Converted to logical properties